### ISSUE TITLE
Deserialization errors when there are Mqtt queues present on the Vhost

### ISSUE THREAD

**Issue by TrevorMare** [EasyNetQ/EasyNetQ.Management.Client#347]
The queue object cannot be deserialized when there are Mqtt queues present on the vhost. 

I suspect that it may be the queue type enum that is failing to deserialize. Attached is an example of the json that is returned via the management Api.

```json
{
        "arguments": {
        },
        "auto_delete": false,
        "durable": true,
        "exclusive": true,
        "name": "mqtt-subscription-direct_command_response_1fbcaqos0",
        "node": "redacted",
        "type": "rabbit_mqtt_qos0_queue",
        "vhost": "marcvhost"
    }
```

I am unsure of when this type is actually set though. I initially thought it may be something with the Qos type but this is not the case.

![image](https://github.com/user-attachments/assets/e2a47f5d-bf62-40e8-bb7f-d4f8e78a1458)

**Comment by zidad** [comment]
Yes that makes sense, if it can contain values like these maybe we need to change it to a string with some constants in the project instead of an enum... Care you create a PR for that?
